ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```
@@ -75,9 +75,9 @@ tests:
       COLLECTION_METHOD: core_bpf
     workflow: stackrox-stackrox-e2e-job
   timeout: 5h0m0s
-- as: gke-nongroovy-e2e-tests
+- always_run: false
+  as: gke-nongroovy-e2e-tests
```

in order to make it always run:
```suggestion
- as: gke-nongroovy-e2e-tests
```

---

@gavin-stackrox It results with `ERROR: One of always_run, run_if_changed or skip_if_only_changed should be set in config`
Should always run be true?

---

Yeah, you are right.